bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd

```diff
@@ -129,6 +130,21 @@
     </xsd:complexType>
 
     <!--
+        Subaru Intensive Program
+    -->
+    <xsd:complexType name="SubaruIntensiveProgramClass">
+        <xsd:complexContent>
+            <xsd:extension base="ProposalClass">
+                <xsd:sequence>
+                    <xsd:element name="telescope" type="ExchangeTelescope"/>
+                    <xsd:element name="submission" type="SubaruIntensiveProgramSubmission"/>
+                </xsd:sequence>
+                <xsd:attribute name="tooOption" type="TooOption" use="optional"/>
```

Comment:
The requirement is that for this type of proposal you select the observatory and that defines the selection of the instruments either Gemini or Subaru